src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
+$cell-size: 100px;
+$digitSize: 30px;
+$gap: 1px;
+$days-of-week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
```

Comment:
The variable `$days-of-week` is defined correctly as a map, which is useful for iterating over days of the week. Ensure that the logic using this map is correctly implemented in the HTML or JavaScript if needed.